Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Components/NodePropertyDisplay/VectorDataInterface.h

```diff
@@ -41,12 +44,16 @@ namespace GraphCanvas
 
         // Returns the truncated display precision for the given index.
         virtual int GetDisplayDecimalPlaces(int /*index*/) const { return 4; }
+
+        virtual void SetSubmitValueEvent(SubmitValueEvent::Handler& handler) { }
+        virtual void OnPressButton() { }
+        virtual AZStd::optional<QPixmap> GetIcon() const { return {}; }
```

Comment:
This does not need to be optional. QPixmap has an isNull function if it is empty, which is the case if default constructed.